store/src/java/com/zimbra/cs/service/mail/SendShareNotification.java

```diff
@@ -114,9 +130,12 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
                     sharesWithGroupGrantee.add(sid);
                 } else {
                     sendNotificationEmail(octxt, mbox, authAccount, account, sid, notes, action, null, null, reqItem);
+                    // create, delete, update 
```

Comment:
We may be able to remove the comment. If it is necessary, please add more information so that it will be meaningful.

Reply:
removed comment as it's defined inside this method